SWLOR.Game.Server/Service/Gui.cs

```diff
@@ -86,6 +86,29 @@ private static void LoadRefreshableViewModels()
             }
         }
 
+        public static void ResetPlayerWindows(uint player)
+        {
+            if (!GetIsPC(player) || GetIsDM(player))
+                return;
+
+            var playerId = GetObjectUUID(player);
+            var dbPlayer = DB.Get<Player>(playerId);
+
+            foreach (var (type, _window) in _windowTemplates)
+            {
+                var window = GetPlayerWindow(player, type);
+                var resizable = JsonObjectGet(_window.Window, "resizable");
+
+                if (JsonGetInt(resizable) == 1)
```

Comment:
Do we care if the window is resizable or not? Couldn't we simply reset it to the initial geometry no matter what?